app/src/main/res/xml/content_settings.xml

```diff
@@ -13,6 +13,13 @@
         app:iconSpaceReserved="false"
         app:useSimpleSummaryProvider="true" />
 
+    <Preference
+        android:key="@string/app_language_android_13_and_up_key"
+        android:title="@string/app_language_title"
+        app:isPreferenceVisible="false"
+        app:singleLineTitle="false"
+        app:iconSpaceReserved="false" />
```

Comment:
I added a separate `Preference` for the Android 13+ behavior only because for some reason when I set an `OnPreferenceClickListener` on the existing preference that returns `true` (indicating "the click was handled"), it still opens the existing dialog.